Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
Thing is that ToLayoutConstraints() is not meant to be called explicitly. 
If we do that, we will be generating a new NSLayoutConstraint object instead of using the one created by the library thus it will be totally useless.
What is the "straightforward workaround" you mention?